components/identifier-generator/back/src/Domain/Model/Delimiter.php

```diff
@@ -5,6 +5,7 @@
 namespace Akeneo\Pim\Automation\IdentifierGenerator\Domain\Model;
 
 /**
+ * String between each element of the identifier
```

Comment:
```suggestion
 * String between each Properties chosen to create the Structure of the identifier
```
That could explain a bit better the link between the classes?
Or maybe juste replace `element` by `propety` to keep the consistency